docs/UserGuide.md

```diff
@@ -118,6 +118,8 @@ Format: `addcontact n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
 
 * A contact can have any number of tags (including 0).
 * `TAG` must be alphanumeric, without spaces.
+* `PHONE_NUMBER` must be a valid Singaporean number, 8 digits long and starting with either 6, 8, or 9.
+* The country code +65 is also accepted at the start of `PHONE_NUMBER`.
```

Comment:
I think adding these tips for `editc` would be nice as well.

Reply:
Added.